### ISSUE TITLE
Bug: Can't toggle selection on item with keyboard navigation (folder only contains one item)

### ISSUE THREAD

**Issue by skst** [files-community/Files#13774]
### Description

When I use the keyboard to open a folder (by selecting a folder and pressing Enter), if there is only one item (file or folder) in that folder, I am unable to use the keyboard to select it. I have tried up/down arrow, Tab, and the Space bar--all of which work in the Windows File Manager.

### Steps To Reproduce

1. Create a folder named "Test."
2. In the "Test" folder, create a file named "Test.txt." (Alternatively, create a folder named "Foo.")
3. Make the "Test" folder the current folder.
4. Use the mouse or keyboard to open the "Test" folder.
5. Try to use the keyboard to select the "Test.txt" file.

### Requirements

Pressing the Up or Down arrow keys or the Space bar should select the file.

### Files Version

3.0.1.0

### Windows Version

10.0.22621.2428

### Log File

[debug.log](https://github.com/files-community/Files/files/13310511/debug.log)

**Comment by yaira2** [comment]
What happens if you type the first letter of the item?

**Comment by skst** [comment]
Sorry, I should have mentioned that, yes, using that method does select the item.

**Comment by yaira2** [comment]
@skst it appears that the shortcut for selecting items is `ctrl + space`, can you see if this works?

**Comment by skst** [comment]
Yes, that does work, but` CTRL+Space` is the shortcut for selecting _multiple_ files/folders, not selecting a single item. (Although, of course, selecting the first item in a multi-selection is the same as selecting a single item.) Space should select the item that has the current focus (and un-select any others). You can try out these shortcuts in Windows File Explorer to confirm that Files does not act the same way. If that's a design choice, that's fine, of course, but I feel that the selection mechanism should match because that's probably what people expect. At least, that's what I was expecting. 🙂

**Comment by D-Maxwell** [comment]
This has been in ready to build for several months now, is it actually going to ship or has it already ?

**Comment by Josh65-2201** [comment]
Ready to build only means some one can work on it. No one has currently, there also isn't a timeline for it.

**Comment by Josh65-2201** [comment]
@skst @D-Maxwell Can you see if this is still an issue in the current preview version. https://files.community/download

**Comment by yaira2** [comment]
This is still an issue

**Comment by yair100** [comment]
This issue will be fixed in the next update
